src/main/java/domain/CarRacing.java

```diff
@@ -0,0 +1,41 @@
+package domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CarRacing {
+
+  private final Cars cars;
+  private static final int MIN_ATTEMPTS = 1;
```

Comment:
```suggestion
  private static final int MIN_ATTEMPTS = 1;

  private final Cars cars;
```

https://dodop-blog.tistory.com/277

이거 참조하면 좋아보여요~

Reply:
감사합니다!